apps/api-server/src/routes/api/project.js

```diff
@@ -226,6 +226,34 @@ router.route('/:projectId') //(\\d+)
 			.catch(next);
 	})
 
+// export a project
+// -------------------
+router.route('/:projectId(\\d+)/export')
+	.all(auth.can('Project', 'view'))
+	.all(function(req, res, next) {
+		const projectId = req.params.projectId;
```

Comment:
Een grote gedeelte van deze functie wordt ook toegepast op andere plekken in deze file. Kan je dit naar een aparte functie trekken die je dan toepast. Dus een functie die een base query accepteerd met de argumenten scopes en includes (beiden arrays) en het resultaat van deze query teruggeeft?